docs/DeveloperGuide.md

```diff
@@ -233,6 +233,12 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Edit instrument feature
+<img src="images/EditInstrumentSequenceDiagram.png" width="600"/>
+<img src="images/EditRefrence1.png" width="600"/>
+<img src="images/EditRefrence2.png" width="600"/>
```

Comment:
<img width="523" alt="ss7" src="https://user-images.githubusercontent.com/60163283/140518743-9f559816-9c17-4a4b-9541-54cf02dcafab.PNG">
No incoming arrow. Its also the wrong method (getParametersToEdit) that's incoming
Can make the activation bar for EditInstrumentParser longer than the sd box. Can look at William and my ref diagram